src/Components/helpers.js

```diff
@@ -1,15 +1,25 @@
-import React, { Fragment, useState } from 'react';
+/* eslint-disable react/prop-types */
```

Comment:
hmm, I believe, it is better not to disable eslint rules. They have some purpose.